clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/FileStoreServiceConfigurator.java

```diff
@@ -74,5 +74,4 @@ public void accept(SingleFileStoreConfigurationBuilder builder) {
         builder.segmented(false)
                 .location(this.pathManager.get().resolveRelativePathEntry(this.relativePath, this.relativeTo))
                 ;
-    }
-}
+    }}
```

Comment:
Nitpick: looks like unintended change.

Reply:
yup

Reply:
BTW can you join the call? I see you are replying on here so perhaps...